pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -249,6 +250,12 @@ func prepareMigrationFlags(isBlockMigration bool) libvirt.DomainMigrateFlags {
 	if isBlockMigration {
 		migrateFlags |= libvirt.MIGRATE_NON_SHARED_INC
 	}
+	val, isSet := os.LookupEnv("UNSAFE_MIGRATION_OVERRIDE")
```

Comment:
How do you set that env variable? It has to be on virt-launcher, right? So it would have to be set in virt-controller? Or do you plan to use a pod preset for that?

Reply:
I was thinking that it should be set using a pod preset, same as LIBVIRT_DEBUG is being set.
This way, it won't be part of the api and will stay hidden, to be used with only specific rare cases.